installer/functions/account/get-account-info.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+
+export const handler: ServerlessFunctionSignature = async function(
+  context: Context,
+  event: {},
+  callback: ServerlessCallback
+) {
+  const client = context.getTwilioClient();
+  const response = new Twilio.Response();
+  response.appendHeader('Access-Control-Allow-Origin', '*');
+  try {
+    const accounts = await client.api.accounts
+      .list({status: 'active', limit: 20})
```

Comment:
why are we fetching 20 accounts when it looks like we only use the first account anyway?